arcgis-ios-sdk-samples/Edit data/Edit and sync features/EditAndSyncFeaturesViewController.swift

```diff
@@ -85,15 +85,18 @@ class EditAndSyncFeaturesViewController: UIViewController {
             if let error = error {
                 self.presentAlert(error: error)
             } else {
-                if let featureServiceInfo = self.geodatabaseSyncTask?.featureServiceInfo,
-                    let map = self.mapView.map {
-                    for index in featureServiceInfo.layerInfos.indices.reversed() {
-                        let layerInfo = featureServiceInfo.layerInfos[index]
-                        // For each layer in the serice, add a layer to the map.
-                        if let layerURL = self.featureServiceURL?.appendingPathComponent(String(index)) {
-                            let featureTable = AGSServiceFeatureTable(url: layerURL)
+                let featureServiceInfo = (self.geodatabaseSyncTask?.featureServiceInfo)!
+                let map = self.mapView.map!
+                for index in featureServiceInfo.layerInfos.indices {
+                    // For each layer in the serice, add a layer to the map.
+                    let layerURL = (self.featureServiceURL?.appendingPathComponent(String(index)))!
+                    let featureTable = AGSServiceFeatureTable(url: layerURL)
+                    _ = featureTable.load { [weak self] error in
```

Comment:
It seems this function has a `Void` return value, rather than an `AGSCancelable` so I wonder if we can safely remove the underscore.

Reply:
I did so because of Phil's[ comment](https://github.com/Esri/arcgis-runtime-samples-ios/pull/964#discussion_r487309775). Did I misunderstand?

Reply:
I think it is safe to change to `featureTable.load { [weak self] error in ...`, as the return value doesn't have any effect on strong reference or have any use. Wait for Phil's comment.

Reply:
Since `load(completion:)` returns `Void`, it really doesn't make sense to ignore the return value. What I was suggesting is that the feature table be made an unowned reference _inside the closure_:
```swift
featureTable.load { [weak self, unowned featureTable] error in
```